src/core/gosling.schema.ts

```diff
@@ -155,9 +157,8 @@ export interface CommonViewDef {
 export type Track = SingleTrack | OverlaidTrack | DataTrack | TemplateTrack;
 
 export interface CommonTrackDef extends CommonViewDef {
-    // !! TODO: we can check if the same id is used multiple times.
-    // !! TODO: this should be track-specific and not defined in views.
-    id?: string; // Assigned to `uid` in a HiGlass view config, used for API and caching.
+    /** Assigned to `uid` in a HiGlass view config, used for API and caching. */
+    id?: string;
```

Comment:
Since `CommonViewDef` also now has an `id?` property, should this be removed?

Reply:
I think I see how the two are different--is my understanding correct? 
- `track.id`:  Set in the schema by the user, or is generated by HiGlass. An instance of GoslingTrackClass knows its `id` by calling `context.viewUid`
- `view.id` Set by the user in the schema or is generated by Gosling